### PR TITLE
Update amp-story-auto-ads spec

### PR DESCRIPTION
**Pull request by lannka**
For #12381

### CODE REVIEW

File: extensions/amp-story/amp-story-auto-ads.md [hunk 1]
```diff
@@ -16,39 +16,75 @@ limitations under the License.
...
-  <tr>
-    <td width="40%"><strong>Examples</strong></td>
-    <td>FILL THIS IN</td>
+    <td>N/A</td>
   </tr>
 </table>
 
 ## Behavior
+`amp-story-auto-ads` extension dynamically inserts ads (implemented as `amp-ad`)
+into the story at the time when the content is consumed.
```
Comment (jasti) [inline extensions/amp-story/amp-story-auto-ads.md]: "While content is being consumed by the user".

File: extensions/amp-story/amp-story-auto-ads.md [hunk 2]
```diff
@@ -16,39 +16,75 @@ limitations under the License.
...
+
+Each `amp-ad` is inserted as a full screen story page. To prevent showing 
+blank/unloaded ads, the ad is pre-rendered completely in the background before 
+being visible to users. Based on user interactions, the extension decides when
```
Comment (jasti) [inline extensions/amp-story/amp-story-auto-ads.md]: "before making it visible to the user"

File: extensions/amp-story/amp-story-auto-ads.md [hunk 3]
```diff
@@ -16,39 +16,75 @@ limitations under the License.
...
+  <amp-story-page>
+  ...
+</amp-story>
+```
 
-FILL THIS IN. What does this extension do?
+`ad-attributes` is a map of key-value pairs, which are the attributes of the 
+ `amp-ad` element to be inserted.
+ 
+The above example will insert the following `amp-ad` element.
```
Comment (jasti) [inline extensions/amp-story/amp-story-auto-ads.md]: "element:"

File: extensions/amp-story/amp-story-auto-ads.md [hunk 4]
```diff
@@ -16,39 +16,75 @@ limitations under the License.
...
+looks like the following:
+
+```html
+<amp-story>
+  <amp-story-auto-ads>
+     <script type="application/json">
+       {
+          "ad-attributes": {
+            "type": "custom",
+            "data-src": "https://adserver.com/getad?slot=abcd1234"
```
Comment (jasti) [inline extensions/amp-story/amp-story-auto-ads.md]: Can you talk a bit about what data-src is? what should publishers do with it?

File: extensions/amp-story/amp-story-auto-ads.md (hunk 1 above)
Comment (lannka) [reply extensions/amp-story/amp-story-auto-ads.md]: done

File: extensions/amp-story/amp-story-auto-ads.md (hunk 2 above)
Comment (lannka) [reply extensions/amp-story/amp-story-auto-ads.md]: done

File: extensions/amp-story/amp-story-auto-ads.md (hunk 3 above)
Comment (lannka) [reply extensions/amp-story/amp-story-auto-ads.md]: done

File: extensions/amp-story/amp-story-auto-ads.md (hunk 4 above)
Comment (lannka) [reply extensions/amp-story/amp-story-auto-ads.md]: added a link to custom ad spec for details.

**Pull request merged**